tools-local/tasks/mobile.tasks/WasmBundleTask/TimeZoneLoader.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Net;
+using System.Reflection;
+using System.Diagnostics;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+public class TimeZoneLoader : Task
+{
+    [Required]
+    public string? InputDirectory { get; set; }
```

Comment:
Why is this parameter needed at all? IIUC, this just seems to be the directory where files are temporarily downloaded. Do they get used after this task is done? If not, then I would suggest removing this, and instead creating a temporary directory, and downloading to that. And removing it at the end of the task.